src/pocketmine/event/entity/EntityDamageEvent.php

```diff
@@ -159,6 +159,7 @@ public function __construct(Entity $entity, $cause, $damage){
 							}
 						}
 					}
+					$this->useArmors();
```

Comment:
But then what editor I use, I have it on my PC and edit with Notepad ++, this change fixed the bug.